app/controllers/admin/deals_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::DealsController < Admin::BaseController
+  before_action :requested_deal, only: [:show, :edit, :destroy, :update]
```

Comment:
rename `requested_deal`. i am not able to find out from the name what the method does

Reply:
renamed to `deal_in_params`